File: backend2/DataAccess/DaoGenerated/システム共有/tt_afokiniiriDto.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: DTO定義
//             お気に入りテーブル
// 作成日　　: 2024.07.23
// 作成者　　: 
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afokiniiriDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afokiniiri";
		public string userid { get; set; }                        //ユーザーID
		public string kinoid { get; set; }                        //機能ID
		public int orderseq { get; set; }                         //並びシーケンス
    }
}

File: GJSBackend/BusinessService/Db/DaoGenerated/システム共有/tt_afokiniiriDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             お気に入りテーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class tt_afokiniiriDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afokiniiri"
        Public Property userid As String                        'ユーザーID
        Public Property kinoid As String                        '機能ID
        Public Property orderseq As Integer                         '並びシーケンス
    End Class
End Namespace
